modules/ROOT/pages/using.adoc

```diff
@@ -95,6 +95,66 @@ $ crc setup # Initialize environment for cluster
 $ crc start # Start the cluster
 ----
 
+[id='about-pullsecrets']
+== About Pull Secrets
+While starting cluster configured with {openshift} or {ushift} preset {prod} would request you to provide a pull secret, it's required to pull some {ocp} specific container images.
+You need to download it from the Pull Secret section of the link:https://console.redhat.com/openshift/create/local[{prod} page on the {rh} Hybrid Cloud Console].
+
+Once you provide this pull secret, it gets stored in Operating System's credential manager so that {prod} doesn't ask for pull secret again in case you delete existing cluster and create
+a new one.
+
```

Comment:
Is this command to find the pull secret location OS agnostic?

```
crc config get pull-secret-file
```

Reply:
(In case CRC has stopped to use this config file: what happens if you have already provisioned this file?)

Reply:
Currently, pull secret can be provided to CRC via these options (in decreasing order of precedence) :
1. Pull Secret from Path `--pull-secret-file`
2. Pull Secret from config `pull-secret-file`
3. Pull Secret from OS Credential Manager

Pull Secret is stored in OS Credential manager only when CRC is not able to resolve it from source 1 and 2. Whenever user enters pull secret via prompt it's stored in the operating system's keyring.

> (In case CRC has stopped to use this config file: what happens if you have already provisioned this file?)

Sorry, I don't understand your question completely. What do you mean by provisioning the file?

Reply:
My question, indeed, makes no more sense with the information you provided. I missed the information that crc has multiple possible sources for the pull secret, with order of precedence (and continue to have them). It might be good to add this info to the docs. (See other comments)